tools/external-adapter/src/index.ts

```diff
@@ -1,16 +1,13 @@
 import express from 'express'
 import bodyParser from 'body-parser'
+import morganBody from 'morgan-body'
 
 const app = express()
-app.use(bodyParser.json())
 
-app.use((req, _, next) => {
-  console.log(`Requested ${req.method} to ${req.path}`)
-  console.log('Request Data:', req.body)
-  next()
-})
+app.use(bodyParser.json())
+morganBody(app)
```

Comment:
Do we really need this dep? We're not using morgan for anything else. Maybe we should just log 
```ts
JSON.stringify(req.body,null,2)
``` 

in our request handlers, we only have two handlers in this case

Reply:
hmm I'm not married to it... but I did find it useful to see both the request and response logged during the test. We could write our own custom middleware to do that, but the process seemed [a bit verbose](https://stackoverflow.com/questions/19215042/express-logging-response-body) when a 1-liner tool already existed.

Reply:
i didnt mean that we should write our own middleware, since we only have 2 handlers in this case it does make sense to just code in the logs directly to avoid having another dep imo

Reply:
ahh okay fair. I'll rework it...